app/src/main/assets/map.html

```diff
@@ -66,13 +66,21 @@
         });
         windLayer.setOpacity(0.7);
 
+        // Clouds
+        var cloudsLayer = new L.TileLayer("http://tile.openweathermap.org/map/clouds_new/{z}/{x}/{y}.png?appid=" + getUrlParameter('appid'), {
+            minZoom: 1,
+            maxZoom: 15
+        });
+        cloudsLayer.setOpacity(0.7);
+         map.addLayer(cloudsLayer);
+
         // Rain
         var rainLayer = new L.TileLayer("http://tile.openweathermap.org/map/precipitation_new/{z}/{x}/{y}.png?appid=" + getUrlParameter('appid'), {
             minZoom: 1,
             maxZoom: 15
         });
         rainLayer.setOpacity(0.7);
-        map.addLayer(rainLayer);
```

Comment:
i think this line is necessary

Reply:
okay, I understand. I fix it

Reply:
this code is added 75 line

Reply:
I don't understand your response. According to Github, you deleted line 75 in the file "map.html". Why, I think it's necessary to have it there?

Reply:
Could you address this please @SIMSEUNGMIN ?

Reply:
In this my code, this code you said is 75 line. 
The reason that I deleted "map.addLayer(rainLayer);" and put new code is the screen shows the cloud map first when you click the map.

Reply:
OK, I'll accept the pull when we're due for a new release in a week or so. Cheers